Sources/SecureXPC/Client/XPCClient.swift

```diff
@@ -13,8 +13,9 @@ import Foundation
 /// There are two different types of services you can communicate with using this client: XPC Services and XPC Mach services. If you are uncertain which
 /// type of service you're using, it's likely an XPC Service.
 ///
-/// **XPC Services**
+/// Clients can also be created from an ``XPCServerEndpoint`` which is the only way to create a client for an anonymous server.
```

Comment:
```suggestion
/// Clients can also be created from an ``XPCServerEndpoint``, which is the only way to create a client for an anonymous server.
```

Reply:
I believe a comma in this case wouldn't be appropriate as the clause which follows is essential information. See https://www.grammarly.com/blog/comma-before-which/.